### PR TITLE
Microverse HTML Form project

### PR DESCRIPTION
**Pull request by sumon0002001**
## Mint Form Clone Microverse Project

In this project we:
- Created a form using HTML and following the correct practices to create a proper form that is up to standards.
- Emulated the styling of the Mint Sign up page using Pure CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                    <a class="learn-more" href="#">Learn more</a>
+                </div>
+            </div>
+            <form action="POST" id="form-input">
+                <div class="container">
+                    <div class="form-group">
+                        <label for="email">
+                                Email Address
+                                <input type="text" name="email" id="email">
+                        </label>
```
Comment (javitocor) [inline index.html]: [OPTIONAL] 
You can add to this input "type: email", check this [link](https://www.w3schools.com/html/html_form_input_types.asp) for more information

**Review by javitocor — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED ♻️ 

Team, great job building the web! 👍 
Good descriptive commits and readme! 👏 
Clean code in both HTML and CSS 💯 

You have met all the requirements for this project 🥇 , however, there is one thing that prevents me from approving the project

Check the comments below 👇

After the required changes are made, please resubmit your project for a new code review ⏩ 

For any question or doubt, you can reach me on Slack

Good luck guys 😃  

The main reason to not approve this project is the sizing, kindly modify the size of the form section and the font-size as well, to make it more similar to the original
Original:
Capture1

Your project:
Capture

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                        <label for="email">
+                                Email Address
+                                <input type="email" name="email" id="email">
+                        </label>
+                    </div>
+                    <div class="form-group">
+                        <label for="phone">
+                                Phone (reccomended)
+                                <input type="number" name="phone" id="phone">
+                        </label>
```
Comment (bertil291utn) [inline index.html]: ## Type tel
- [x] It has an `<input>` of type `tel` and a label `Phone` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 10px 30px;
+  font-size: 1.1em;
+}
+
+.acc-info {
+  display: flex;
```
Comment (bertil291utn) [inline styles/style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```CSS
.flex{
  display: flex;
}
```
**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: styles/style.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+#main-form {
+  background-color: white;
+  width: 700px;
+  margin: 0 auto;
+  border: 0.5px solid lightgray;
+}
+
+.form-center {
+  text-align: center;
+}
```
Comment (bertil291utn) [inline styles/style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```CSS
.text-align-center{
   text-align: center;
}
```
**Html file**

```HTML
<div class="text-align-center">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:
General requirements 
- [x] Edit pull request message following this [guideline](https://cdn.filestackcontent.com/osvFJEkQHWQsgDe0PJoQ)

## OPTIONAL
* [x] Please use of the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
* [x] Please use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Summary
- Please avoid using the same repetitively class, create only once and use it 
-  Don't forget to work with inline comments below this comment box.

***

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by teekaytech — APPROVED** [review]
# Status : Approved :heavy_check_mark: 

### Hello team, 

**You've done an awesome job :clap: 
Your project passed all the requirements.**  :muscle: 

**[COMMENTS]**

- [ ] There's a weird button fixed at the bottom right corner of your page named `Privacy Settings`. I'm not sure of where this comes from, but I'm definitely sure it's not from the original page. See the image below for better understanding:

Your page: 

l

Original Page:


k

I think it's not part of the page. Kindly remove it.

**Once again, congratulations! :tada: 
It's a pleasure to review your project :blush: 
Good luck with your next project :100: 
Happy Coding :man_technologist:**

_With all my regards,_
Taofeek Olalere
@taofeekolalere {slack}
@teekaytech {github}

**Pull request merged**